bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/connection/HueBridge.java

```diff
@@ -106,35 +147,26 @@ public HueBridge(String ip, int port, String protocol, ScheduledExecutorService
      * Use the ip only constructor and authenticate() function if
      * you don't want to connect right now.
      *
+     * @param httpClient instance of the Jetty shared client
      * @param ip ip address of bridge
      * @param port port of bridge
      * @param protocol protocol to connect to the bridge
      * @param username username to authenticate with
+     * @param scheduler the ExecutorService to schedule commands
      */
-    public HueBridge(String ip, int port, String protocol, String username, ScheduledExecutorService scheduler)
-            throws IOException, ApiException {
-        this(ip, port, protocol, scheduler);
+    public HueBridge(HttpClient httpClient, String ip, int port, String protocol, String username,
+            ScheduledExecutorService scheduler) throws IOException, ApiException {
```

Comment:
Other exceptions can be thrown by the call of method authenticate : ConfigurationException, UnauthorizedException

Reply:
I've checked that they were added.